state/execution.go

```diff
@@ -388,7 +388,7 @@ func (blockExec *BlockExecutor) Commit(
 	)
 
 	// Update mempool.
-	err = blockExec.mempool.Update(
+	blockExec.mempool.Update(
 		block.Height,
 		block.Txs,
 		abciResponse.TxResults,
```

Comment:
Minor: can you please use `nil`, rather than `err` in line 399 below to improve readability?